src/accelerate/utils.py

```diff
@@ -51,7 +51,36 @@ def is_sagemaker_available():
 OPTIMIZER_NAME = "optimizer"
 
 
-class RNGType(Enum):
+class MetaEnum(EnumMeta):
```

Comment:
```suggestion
class EnumWithContains(EnumMeta):
```
nit: the name is not very descriptive on what the metaclass does, would switch to this.